index.js

```diff
@@ -36,11 +41,26 @@ module.exports = function loadIPSet (input, opts, cb) {
       .on('error', cb)
       .pipe(split())
       .on('data', function (line) {
-        var match = blocklistRe.exec(line)
-        if (match) blocklist.push({start: match[2], end: match[3]})
+        var match = ipSetRegex.exec(line)
+        if (match) {
+          blocklist.push({start: match[2], end: match[3]})
+        } else {
+          match = ipv4NetSetRegex.exec(line)
+          if (match) blocklist.push(parseIPRange(match))
+        }
       })
       .on('end', function () {
         cb(null, new IPSet(blocklist))
       })
   }
+
+  function parseIPRange (regexMatch) {
+    var ip = regexMatch[2]
+    var bitMask = regexMatch[3]
+
+    var ipRange = ip + '/' + bitMask
+    var range = new Netmask(ipRange)
+
+    return {start: range.first, end: range.last}
```

Comment:
we should use `range.broadcast` because that's what this library is calling the real last IP.